lib/src/view/opening_explorer/opening_explorer_screen.dart

```diff
@@ -0,0 +1,845 @@
+      ),
+    );
+  }
+}
+
+class _Opening extends StatelessWidget {
+  const _Opening({required this.opening});
+
+  final Opening opening;
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.only(left: 6.0),
+      color: Theme.of(context).colorScheme.primaryContainer,
+      child: Row(
+        children: [
+          if (opening.eco.isEmpty)
+            Text(opening.name)
+          else
+            Text('${opening.eco} ${opening.name}'),
+        ],
+      ),
+    );
+  }
+}
+
+class _MoveTable extends ConsumerWidget {
+  const _MoveTable({
+    required this.moves,
+    required this.whiteWins,
+    required this.draws,
+    required this.blackWins,
+    required this.ctrlProvider,
+  });
+
+  final IList<OpeningMove> moves;
+  final int whiteWins;
+  final int draws;
+  final int blackWins;
+  final AnalysisControllerProvider ctrlProvider;
+
+  String formatNum(int num) => NumberFormat.decimalPatternDigits().format(num);
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    const rowPadding = EdgeInsets.all(6.0);
+    final games = whiteWins + draws + blackWins;
+
+    return Table(
+      columnWidths: const {
+        0: FractionColumnWidth(0.2),
+        1: FractionColumnWidth(0.3),
+        2: FractionColumnWidth(0.5),
+      },
+      children: [
+        TableRow(
+          decoration: BoxDecoration(
+            color: Theme.of(context).colorScheme.primaryContainer,
+          ),
+          children: [
+            Container(
+              padding: rowPadding,
+              child: Text(context.l10n.move),
+            ),
+            Container(
+              padding: rowPadding,
+              child: Text(context.l10n.games),
+            ),
+            Container(
+              padding: rowPadding,
+              child: Text(context.l10n.whiteDrawBlack),
+            ),
+          ],
+        ),
+        ...List.generate(
+          moves.length,
+          (int index) {
+            final move = moves.get(index);
+            final percentGames = ((move.games / games) * 100).round();
+            return TableRow(
+              decoration: BoxDecoration(
+                color: index.isEven
+                    ? Theme.of(context).colorScheme.surfaceContainerLow
+                    : Theme.of(context).colorScheme.surfaceContainerHigh,
+              ),
+              children: [
+                TableRowInkWell(
+                  onTap: () => ref
+                      .read(ctrlProvider.notifier)
+                      .onUserMove(Move.fromUci(move.uci)!),
+                  child: Container(
+                    padding: rowPadding,
+                    child: Text(move.san),
+                  ),
+                ),
+                TableRowInkWell(
+                  onTap: () => ref
+                      .read(ctrlProvider.notifier)
+                      .onUserMove(Move.fromUci(move.uci)!),
+                  child: Container(
```

Comment:
Padding. I won't put the comment again but should be applied to all `Container`.